todo/todo.py

```diff
@@ -183,344 +173,370 @@ async def showsettings(self, ctx):
         try:
-            pred = MessagePredicate.yes_or_no(ctx)
-            result = await self.bot.wait_for("message", check=pred, timeout=15.0)
+            await self.bot.wait_for("message", check=pred)
         except asyncio.TimeoutError:
-            await ctx.send("Alright!")
-            reverse = True
+            await ctx.send("Okay, I won't sort them in reverse")
+            result = False
         else:
-            reverse = pred.result
-        await self.config.user(ctx.author).reverse_sort.set(reverse)
-        async with self.config.user(ctx.author).todos() as items:
-            items.sort(reverse=reverse)
-            await ctx.send("Okay! I've sorted your todos!")
-            await self.page_logic(
-                ctx=ctx, things=[f"{num}. {i}" for num, i in enumerate(items, 1)]
-            )
+            result = pred.result
+        await ctx.message.add_reaction("\N{WHITE HEAVY CHECK MARK}")
+        await self.config.user(ctx.author).reverse_sort.set(result)
+        async with self.config.user(ctx.author).completed() as todos:
+            todos.sort(reverse=result)
 
-    @todo.command(
-        aliases=[
-            "move",
-        ]
-    )
-    async def rearrange(self, ctx, index: positive_int, new_index: positive_int):
-        """Rearrange a todo!"""
-        index -= 1
-        new_index -= 1
-        async with self.config.user(ctx.author).todos() as items:
-            if new_index > (leng := len(items)):
-                new_index = leng - 1
-            try:
-                item = items.pop(index)
-            except IndexError:
-                await ctx.send("Hm, there doesn't seem to be a todo there")
-            else:
-                items.insert(new_index, item)
-        await ctx.send(f"Moved todo reminder `{item}`")
-        await self.config.user(ctx.author).autosort.set(False)
-
-    @todo.command()
-    async def show(self, ctx, index: positive_int):
-        """Show a todo reminder by index!"""
-        act_index = index - 1
+    @complete.command(name="list")
+    async def complete_list(self, ctx):
+        """List your completed todos"""
+        completed = await self.config.user(ctx.author).completed()
+        if not completed:
+            await ctx.send(_no_completed_message.format(prefix=ctx.clean_prefix))
+        else:
+            if not await self.config.user(ctx.author).use_md():
+                completed = await self._cross_lists(completed)
+            completed = await self._number_lists(completed)
+            await self.page_logic(ctx, completed, "Completed todos")
+
+    @complete.command(name="remove", aliases=["del", "rm"])
+    async def complete_remove(self, ctx, *indexes: positive_int):
+        """Remove compeleted todos"""
+        if not indexes:
+            return await ctx.send_help(ctx.command)
+        indexes = [x - 1 for x in indexes]
+        _ = await self.config.user(ctx.author).completed()
+        if not _:
```

Comment:
Yes, you used to tell me off for the way I named my variables, so I'm doing it back 😄
Perhaps change this to something better from the code's perspective.

```suggestion
        completed = await self.config.user(ctx.author).completed()
        if not completed:
```